models/health_config.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+// Annotationkey is a mnemonic type name for string
+type AnnotationKey string
+
+const (
+	AllHealthAnnotation  AnnotationKey = ".*"
+	RateHealthAnnotation AnnotationKey = "health.kiali.io/rate"
+)
+
+func GetHealthConfigAnnotation() []AnnotationKey {
+	return []AnnotationKey{RateHealthAnnotation}
+}
+
+func GetHealthAnnotation(annotations map[string]string, filters []AnnotationKey) interface{} {
```

Comment:
Another comment, I looked close to this code.

I guess this signature:

```
func GetHealthAnnotation(annotations map[string]string, filters []AnnotationKey) interface{} {
```

Could be simplified as "filters" is always a constat in the current use case (if I looked the code correctly), so, I'd remove the "filters []AnnotationKey" and I'd use it internally of the method as a constant.

When this annotations feature grows, it can be refactored, but for now I guess it will help to simplify it.